src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -322,10 +334,12 @@ private async Task<(Document, bool addedFields)> GenerateFieldDelegatingConstruc
                 Dictionary<string, ISymbol> parameterToExistingFieldMap,
                 Dictionary<string, string> parameterToNewFieldMap,
                 bool caseSensitive,
-                out ImmutableArray<ParameterName> newParameterNames)
+                out ImmutableArray<ParameterName> newParameterNames,
+                NamingRule fieldNamingRule)
```

Comment:
💡 Would be better to reorder this so the `out` parameter is last

Reply:
Fixed